config/settings/base.py

```diff
@@ -216,3 +239,6 @@
 API_BASE_URL = env('API_BASE_URL')
 # CONN_MAX_AGE=600
 
+DEFAULT_AUTO_FIELD = 'django.db.models.BigAutoField'
+
+USER_SECRET_KEY = ["secret"]
```

Comment:
will this be an array

Reply:
TBH, I don't know lol, let me just remove it.,